lhotse/bin/modes/cut.py

```diff
@@ -44,6 +45,27 @@ def simple(
     cut_set.to_yaml(output_cut_manifest)
 
 
+@cut.command()
+@click.argument('feature_manifest', type=click.Path(exists=True, dir_okay=False))
+@click.argument('output_cut_manifest', type=click.Path())
+@click.option('-d', '--cut-duration', type=float, default=5.0, help='How long should the cuts be in seconds.')
+@click.option('-s', '--cut-shift', type=float, default=None,
+              help='How much to shift the cutting window in seconds (by default the shift is equal to CUT_DURATION).')
+def windowed(
```

Comment:
I'll extend this with an option to keep/discard "windows" that are shorter than `cut_duration` (and maybe a separate option to choose if they should be padded or not).